src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/WorkflowRunController.kt

```diff
@@ -136,4 +147,16 @@ class WorkflowRunController(
                 WorkflowRunStatus.WorkflowRunReportStatus(reportName, report.key, report.status, report.version)
             })
     }
+
+    fun getWorkflowRunReportByWorkflowKey(): List<WorkflowRunReport>
```

Comment:
```suggestion
    fun getWorkflowRunReporstByWorkflowKey(): List<WorkflowRunReport>
```

Reply:
Could you explain what these methods, `getWorkflowRunReportByWorkflowKey` and `getWorkflowRunReportByReportKey` are for on the controller? There aren't any new endpoints, and nothing seems to call these methods